glide.yaml

```diff
@@ -22,8 +22,10 @@ import:
 - package: gopkg.in/tchap/go-patricia.v2
   subpackages:
   - patricia
+- package: golang.org/x/text
+  version: 19e51611da83d6be54ddafce4a4af510cb3e9ea4
 - package: k8s.io/client-go
-  version: 243d8a9cb66a51ad8676157f79e71033b4014a2a
+  version: release-3.0
```

Comment:
Exact versions are locked in `glide.lock`, unless it is deleted exact version will be downloaded during build. Usually when `glide.lock` is deleted is to update dependencies while staying within constraints of versions specified in `glide.yaml`. 

In case of client-go, given their branching policy, using a branch is a safe bet, especially given that many other deps are in `glide.yaml` are not locked (==locked to master)